litmus/director/openebs-install/run_litmus_test.yml

```diff
@@ -0,0 +1,93 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: openebs-install
```

Comment:
Put a `-` after openebs-install so that the object is created as `openebs-install-xxxx` else it will be created as `openebs-installxxxx` which will not be appropriate.